bubblewrap.c

```diff
@@ -2213,13 +2248,16 @@ main (int    argc,
       /* Optionally bind our lifecycle to that of the parent */
       handle_die_with_parent ();
 
-      if (opt_info_fd != -1)
+      if (opt_info_fd != -1 || opt_json_status_fd != -1)
         {
           cleanup_free char *output = xasprintf ("{\n    \"child-pid\": %i\n}\n", pid);
```

Comment:
It seems like it might help parsers if we only emitted a single newline at the end of the document:
https://stackoverflow.com/questions/6886283/how-i-can-i-lazily-read-multiple-json-values-from-a-file-stream-in-python

I think I suggested this "multiple json object stream" and I'm sure I saw it somewhere else...ah wait I just remembered, the [Docker "events" API](https://docs.docker.com/engine/api/v1.24/) does this.

So...hopefully people won't have trouble parsing it with various libraries.

Reply:
It wouldn't have been too difficult to parse multi-line objects, since it's easy enough to merge multiple lines.
The only real difficulty would have been if you could have multiple objects without a clear delimiter character.
I've used "JSON lines" in other projects before, and having one object per line is much easier to parse.